### DISCUSSION TITLE
Feature : Twitch unskippable ads

### DISCUSSION THREAD

**Discussion by NRWaldik** [night/betterttv#5956] [Ideas]
I don't if it's in your scope but currently Twitch is hard bombing people with 15-30sec ads and it's just too much tbh

I just want to remove these specific ads and whether chose when I want or don't want to watch them

Thank you for reading me

**Comment by night** [comment]
Thanks for the suggestion. BetterTTV used to block ads long ago, but we unfortunately do not block Twitch advertisements any longer (for many years now) due to prior warnings given to us by Twitch.
